package.json

```diff
@@ -91,12 +95,9 @@
     "@types/testing-library__jest-dom": "^5.9.1",
     "@typescript-eslint/eslint-plugin": "^5.59.1",
     "@vitejs/plugin-react": "^4.0.0",
-    "babel-jest": "^27.4.2",
-    "babel-preset-react-app": "^10.0.1",
-    "blob-polyfill": "^7.0.20220408",
```

Comment:
Many test-related dependencies have been moved to `@terra-ui-packages/test-utils`.